src/Compilers/Core/Portable/Symbols/Attributes/AttributeDescription.cs

```diff
@@ -538,6 +539,7 @@ static AttributeDescription()
         internal static readonly AttributeDescription AssemblyAlgorithmIdAttribute = new AttributeDescription("System.Reflection", "AssemblyAlgorithmIdAttribute", s_signaturesOfAssemblyAlgorithmIdAttribute);
         internal static readonly AttributeDescription DeprecatedAttribute = new AttributeDescription("Windows.Foundation.Metadata", "DeprecatedAttribute", s_signaturesOfDeprecatedAttribute);
         internal static readonly AttributeDescription NullableAttribute = new AttributeDescription("System.Runtime.CompilerServices", "NullableAttribute", s_signaturesOfNullableAttribute);
+        internal static readonly AttributeDescription NullableMembersAttribute = new AttributeDescription("System.Runtime.CompilerServices", "NullableMembersAttribute", s_signaturesOfNullableMembersAttribute);
```

Comment:
>NullableMembersAttribute [](start = 54, length = 24)

It doesn't looks like we pay attention to this attribute on import. Are we using this description anywhere? Perhaps we should disallow an explicit attribute application in source.

Reply:
Logged [https://github.com/dotnet/roslyn/issues/36457](https://github.com/dotnet/roslyn/issues/36457).

---
In reply to: [293557364](https://github.com/dotnet/roslyn/pull/36398#discussion_r293557364) [](ancestors = 293557364)

Reply:
> Logged [https://github.com/dotnet/roslyn/issues/36457](https://github.com/dotnet/roslyn/issues/36457).

Would it make sense to revert changes in this file if they are not used/tested.

---
In reply to: [293911855](https://github.com/dotnet/roslyn/pull/36398#discussion_r293911855) [](ancestors = 293911855,293557364)

Reply:
This `AttributeDescription` is used by `PEAssemblyBuilder` in determining whether to synthesize the type.

---
In reply to: [293917589](https://github.com/dotnet/roslyn/pull/36398#discussion_r293917589) [](ancestors = 293917589,293911855,293557364)